### PR TITLE
IRONSIDE-1166 -- final merge of latest release, release/connector/1.3.0 to master.

### PR DESCRIPTION
**Pull request by alvinkrebs**
**Merge release 1.3.0 to master.**

1. Several modifications to the rticonnextdds-connector. The largest being we no longer commit libraries. So, you'll see the lib/ directory disappear.
2. Supporting this change are various configuration changes and new scripts. README and LICENSE were also adjusted.

### CODE REVIEW

**Review by elainerti — APPROVED** [review]

**Review by elainerti — COMMENTED** [review]
Validation seems to fail. @ManuelJNunez  any idea how to address this?

**Review by ManuelJNunez — APPROVED** [review]

**Comment by ManuelJNunez** [comment]
> Validation seems to fail. @ManuelJNunez  any idea how to address this?

This is because of a backport on Node that caused problems with one of our Third-Party dependencies. This was already addressed on develop.

**Comment by ManuelJNunez** [comment]
> Validation seems to fail. @ManuelJNunez  any idea how to address this?

One option is to modify the ci_config.yaml to use the version that worked correctly before the last Node update.

File: ci_config.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+node_versions:
+  - "18"
```
Comment (ManuelJNunez) [inline ci_config.yaml:2]: 18.19 should work

File: ci_config.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  - "19"
+  - "20"
```
Comment (ManuelJNunez) [inline ci_config.yaml:4]: 20.11 should work

**Review by ManuelJNunez — CHANGES_REQUESTED** [review]

File: ci_config.yaml (hunk 2 above)
Comment (alvinkrebs) [reply ci_config.yaml:4]: I don't see these values in develop or release/connector/1.3.0.

Comment (ManuelJNunez) [reply ci_config.yaml:4]: Yes, but I think that it is better to change it than seeing master (a branch that should be stable and the first that you see when you open the repository) with a failing CI

Comment (Alxe) [reply ci_config.yaml:4]: For reference, `release/connector/1.3.0` doesn't have #203 (CON-304) applied, thus CI will fail as @ManuelJNunez said.

I agree we should narrow our Node versions used in CI to the ones that we know will work. We can continue the discussion offline on whether `master` makes sense or not.

@alvinkrebs Based on #198, we should have these versions be:
```
node_versions:
  - "18.19"
  - "19"
  - "20.11"
```

19 is not a LTS release, so the change shouldn't have been backported but I've not confirmed this.

**Comment by alvinkrebs** [comment]
Since we are changing source, we need a feature branch.

**Pull request closed without merging**